roles/keystone/meta/main.yml

```diff
@@ -44,3 +44,6 @@ dependencies:
       - { protocol: tcp, port: "{{ endpoints.keystone.port.haproxy_api }}" }
       - { protocol: tcp, port: "{{ endpoints.keystone_admin.port.haproxy_api }}" }
       - { protocol: tcp, port: "{{ endpoints.keystone_legacy.port.haproxy_api }}" }
+  - role: repos
+    repo: opensuse
```

Comment:
We should not use opensuse for rdo. We should open up support case on redhat to see which repo provided needed rpm.